Packs/Coralogix/Integrations/Coralogix.py

```diff
@@ -0,0 +1,447 @@
+        cgx_endpoint_url - String, Coralogix Cluster Base URL for the Coralogix ES-API
+        application_name - String, The
+        subsystem_name - String,
+        severity - String
+
+    Returns:
+        Returns the incidents found in Coralogix
+    """
+
+    last_run_timestamp = -1
+    if "last_run_timestamp" in demisto.getLastRun():
+        last_run_timestamp = demisto.getLastRun()["last_run_timestamp"]
+    raw_data = clx_search_command(
+        cgx_private_key=cgx_private_key,
+        cgx_endpoint_url=cgx_endpoint_url,
+        query=incidents_base_query,
+        application_name=application_name,
+        subsystem_name=subsystem_name,
+        severity=severity,
+        exclude_columns=[],
+        results_as_table=False,
+        since_timestamp=last_run_timestamp
+    )
+
+    newest_incident_date_obj = datetime.datetime(year=1970, month=1, day=1)
+    incidents = []
+    if "hits" in raw_data and "hits" in raw_data["hits"]:
+        for document in raw_data["hits"]["hits"]:
+            flattened_document = flatten_json(document["_source"])
+            incident_date = flattened_document['@timestamp']
+            incident = {
+                'name': flattened_document[incidents_name_field],
+                'occurred': incident_date,
+                'rawJSON': json.dumps(flattened_document)
+            }
+            incidents.append(incident)
+            incident_date_obj = dateutil.parser.parse(incident_date)
+            if incident_date_obj.replace(tzinfo=datetime.timezone.utc).timestamp() > newest_incident_date_obj.replace(tzinfo=datetime.timezone.utc).timestamp():
+                newest_incident_date_obj = incident_date_obj
+
+    demisto.setLastRun({"last_run_timestamp": newest_incident_date_obj.replace(tzinfo=datetime.timezone.utc).timestamp()})
+    return incidents
+
+def main():
+    """
+        PARSE AND VALIDATE INTEGRATION PARAMS
+    """
+    private_key = demisto.params()['privatekey'].strip()
+
+    # get the service API url
+    tags_api_url = urljoin(demisto.params()['webapi_url'].strip(), '/api/v1/addTagPost')
+    search_api_url = urljoin(demisto.params()['esapi_url'].strip(), '/*/_search')
+
+    tag_application_name = demisto.params()['app_name'].strip()
+    tag_subsystem_name = demisto.params()['subsystem_name'].strip()
+
+    if 'incidents_query' in demisto.params() and demisto.params()['incidents_query'] is not None:
+        incidents_query = demisto.params()['incidents_query'].strip()
+    else:
+        incidents_query = "alert_type:\"53d222e2-e7b2-4fa6-80d4-9935425d47dd\""
+    if 'incidents_application_name' in demisto.params() and demisto.params()['incidents_application_name'] is not None:
+        incidents_application_name = demisto.params()['incidents_application_name'].strip()
+    else:
+        incidents_application_name = ""
+    if 'incidents_subsystem_name' in demisto.params() and demisto.params()['incidents_subsystem_name'] is not None:
```

Comment:
I don't see this parameter in your yml file

Reply:
Added now